src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -400,11 +400,26 @@ public void writeJavaDocCommentStatement_allComponents() {
   }
 
   @Test
-  public void writeBlockComment_specialChar() {
-    String content = "Testing special characters: \b\t\n\r\"`'?/\\,.[]{}|-_!@#$%^()";
+  public void writeBlockComment_shortLines() {
+    String content = "Apache License \nThis is a test file header";
```

Comment:
Nit: Could we please reuse the `createLines` utility to make the expected content more clear?